pallets/registrar/src/benchmarks.rs

```diff
@@ -197,10 +242,15 @@ mod benchmarks {
     }
 
     #[benchmark]
-    fn deregister_scheduled(x: Linear<5, 3_000_000>, y: Linear<1, 50>) {
-        let storage = vec![(b"code".to_vec(), vec![1; x as usize]).into()];
-        let storage = new_genesis_data(storage);
-        let genesis_para_id_len = Pallet::<T>::registered_para_ids().len();
+    fn deregister_scheduled() {
+        let x = T::MaxGenesisDataSize::get();
+        let y = T::MaxLengthParaIds::get();
+        let storage = max_size_genesis_data::<T>(1, x);
+        let genesis_para_id_len = 0;
```

Comment:
Is this needed?

Reply:
Forgot to remove it, this will always be 0